Makefile

```diff
@@ -1,3 +1,10 @@
 build-wrapper:
 	cd wrapper && cargo build --release --target wasm32-wasip1
 	cp wrapper/target/wasm32-wasip1/release/wrapper.wasm descriptors/
+
+check-wrapper-compiled: build-wrapper
+	@$(call print, "Verifying wrapper is up to date.")
+	if test -n "$$(git status --porcelain '*.go')"; then echo "WASM wrapper not compiled!"; git status --porcelain '*.go'; exit 1; fi
```

Comment:
What is the purpose of this line? I am not sure I understand it. The name and comment say it verifies the wrapper is up to date, but `build-wrapper` creates a `.wasm` file, and does not touch .go files?

Reply:
Oops, just copy/paste without looking closely... Changed to `*.wasm`.